paluchas-durham.md

```diff
@@ -0,0 +1,10 @@
+# Durham, UK
+
+## Places to Eat
+- Lebabeat
+- The Cellar Door
```

Comment:
Lebaneat serves nice spicy middle east food. The Cellar Door serves nice clasics.